site/styles/main.css

```
@@ -0,0 +1,148 @@
+* {
+    box-sizing: border-box;
+  }
+
+  body {
+    background-color: #E7E6DB;  
+    color: #3A3B47;             
+    font-family: Arial, sans-serif;
+  }
+
+  .width-wrap {
+    background-color: #A3A56E; 
+    max-width: 960px;
+    margin: 0 auto 3em;
+    padding: 0 2em 2em; 
+    border: 1px solid rgba(0, 0, 0, 0.3);
+    border-top: none;
+    position: relative;
+  }
```

1. `Lab 5: Styling > basic styles > width-wrap elements has the style > border with > black with 0.3 alpha`
> Expected string: "rgb(0, 0, 0) rgba(0, 0, 0, 0.3) rgba(0, 0, 0, 0.3)"
> Received string: "rgb(58, 59, 71) rgba(0, 0, 0, 0.3) rgba(0, 0, 0, 0.3)"

2. `Lab 5: Styling > basic styles > width-wrap elements has the style > border with > no top border`
>    Expected string: "0px none rgb(0, 0, 0)"
>    Received string: "0px none rgb(58, 59, 71)"

This failure is a little insidious. This is failing due to the order in which properties were defined. To fix it you would need to do the following:
```suggestion
  .width-wrap {
    max-width: 960px;
    margin: 0 auto 3em;
    padding: 0 2em 2em; 
    border: 1px solid rgba(0, 0, 0, 0.3);
    border-top: none;
    position: relative;
    background-color: #A3A56E; 
  }
```

Because you set the background-colour first the border-top-color was set, which led to a top border being found.